oak_functions_service/src/logger.rs

```diff
@@ -14,8 +14,6 @@
 // limitations under the License.
 //
 
-#![no_std]
-
```

Comment:
slightly unfortunate consequence that these are now no longer no std compatible

Reply:
Quite the opposite, now everything in the crate is by necessity no_std compatible

Reply:
oh nice